shell/lib/db-deprecated.js

```diff
@@ -74,6 +74,7 @@ if (Meteor.isServer) {
 if (Meteor.isClient) {
   Session.setDefault("shrink-navbar", false);
   globalGrains = new ReactiveVar([]);
+  Session.set("shrink-navbar", window.localStorage.getItem("shrink-navbar") === "true");
```

Comment:
+1 to this getting merged

One day I might make it reactive to other tabs, but I think it's not a
showstopper.